src/current/v24.1/cost-based-optimizer.md

```diff
@@ -359,18 +362,20 @@ For a join hint example, see [Use the right join type]({% link {{ page.version.v
 
 ### Supported join algorithms
 
-- `HASH`: Forces a hash join; in other words, it disables merge and lookup joins. A hash join is always possible, even if there are no equality columns - CockroachDB considers the nested loop join with no index a degenerate case of the hash join (i.e., a hash table with one bucket).
+- `HASH`: Forces a hash join; in other words, it disables merge and lookup joins. A hash join is always possible, even if there are no equality columns: CockroachDB considers the nested loop join with no index a degenerate case of the hash join (i.e., a hash table with one bucket).
```

Comment:
```suggestion
- `HASH`: Forces a hash join, disabling merge and lookup joins. A hash join is always possible, even if there are no equality columns: CockroachDB considers the nested loop join with no index a degenerate case of the hash join (i.e., a hash table with one bucket).
```
I find the last sentence incomprehensible, but perhaps your audience understands what it means. "Degenerate" is quite a word.

Reply:
Fixed.